ci-operator/jobs/openstack-k8s-operators/cinder-operator/openstack-k8s-operators-cinder-operator-master-presubmits.yaml

```diff
@@ -63,7 +63,7 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )cinder-operator-build-deploy,?($|\s.*)
   - agent: kubernetes
-    always_run: true
+    always_run: false
```

Comment:
This should be flipped, the `build-deploy-kuttl` should have `always_run: true` and the `build-deploy` should have it `false`